package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@oat-sa/tao-item-runner",
+    "version": "0.1.0",
+    "displayName": "TAO Item Runner",
```

Comment:
Could you please add the `descritption` field. Without it `npm install` will display a warning